tests/vault_test.go

```diff
@@ -256,3 +256,57 @@ func (s *VaultSuite) TestWithdrawToProtected() {
 		s.vault.WithdrawTo(signer(receiver), s.erc20Addresses[0], val, receiver.address()),
 	)
 }
+
+///
+func (s *VaultSuite) TestUpgrade() {
+	receiver := s.account[2]
+	newKey := s.account[3]
+
+	// Deploy the new vault.
+	newVaultAddress, tx, newVault, err := abi.DeployVaultV2(signer(newKey), s.node)
+
+	newVault.ChangeManager(signer(newKey), s.managerAddress)
```

Comment:
These transactions aren't checked for success or failure, and we should probably remove comments